products/paas/subscription.md

```diff
@@ -234,60 +234,164 @@ where "**K**" means 1 thousand, "**M**" means 1 million, "**B**" means 1 billion
   <tbody>
       <tr>
           <td>All Transport messages (Tenant)</td>
-          <td>200/sec<br>but no more then 6K/min<br>but no more then 140K/hour</td>
-          <td>2K/sec<br>but no more then 60K/min<br>but no more then 1.4M/hour</td>
-          <td>10K/sec<br>but no more then 300K/min<br>but no more then 6M/hour</td>
-          <td>20K/sec<br>but no more then 600K/min<br>but no more then 12M/hour</td>
+          <td>200/sec,<br>but less than 6K/min,<br>but less than 140K/hour</td>
```

Comment:
Use the following description style:
up to 200 per second, not exceeding 6,000 per minute, and capped at 140,000 per hour. 
Reference: https://chat.openai.com/share/e/e98f326f-dff8-472a-8e8b-e6a56b027fe9

Reply:
Used the suggested description style for the "Rate Limits" section